packages/components/Callout/package.json

```diff
@@ -30,7 +30,6 @@
   },
   "dependencies": {
     "@fluentui-react-native/adapters": "workspace:*",
-    "@fluentui-react-native/component-cache": "workspace:*",
```

Comment:
Mayyybe worth removing `component-cache` in a separate PR to keep this change smaller?